src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1579,6 +1588,11 @@ internal string GetNameSpace()
                 if (m_namespace == null)
                 {
                     Type type = m_runtimeType;
+
+                    // Since Function pointers don't have a TypeDef metadata record just use the namespace for System.Type.
+                    if (type.IsFunctionPointer)
+                        return typeof(Type).Namespace!;
+
```

Comment:
Why not an empty namespace instead? System is an odd choice.

Reply:
Why do we match the namespace of `int*`? Pointers just take the namespace of whatever is the element type. Which kind of makes sense, but it is also an odd question to ask. We could return the namespace of the return type but that's also very odd.

If there's a precedent for `null`, that sounds like an even better choice.

Reply:
I'll change to `null` - thanks